test/benchmark/sk_benchmark.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: sklearn-load-test
+spec:
+  backoffLimit: 6
+  parallelism: 1
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      containers:
+        - args:
+            - vegeta -cpus=1 attack -duration=10m -rate=5/1s -targets=/var/vegeta/cfg
+              | vegeta report -type=text
```

Comment:
I think its best to rerun these tests again with `-keepalive=false` as I have found for calling internal SVCs you won't get proper round-robin as Vegeta will be reusing TCP connections by default.

Reply:
I think in the real case user also want `keepalive=true` so vegeta default is good.

Reply:
But in the real case users will be usually calling via an Ingress which will ensure load balancing for downstream connections via say Envoy. I think the issue is that you may find in the HPA case that some pods get overloaded when internal SVC calls from Vegeta reuse TCP connections.

Reply:
For the HPA case if you check the setup above I also create a virtual service which routes to the internal svc, so it is going through the local cluster gateway and envoy is doing the load balancing. The comparison should be fair since both are hitting from the cluster local gateway, for HPA case it directly routes to the user container from kubernetes internal svc while for KPA case it is gateway -> activator -> queue-proxy-> user container.